packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -300,8 +305,15 @@ export const prepareAuctionBook = (baggage, zcf) => {
          *  @param {Ratio} bidScaling
          *  @param {Amount<'nat'>} want
          *  @param {boolean} trySettle
+         *  @param {boolean} [exitAfterBuy]
          */
-        acceptScaledBidOffer(seat, bidScaling, want, trySettle) {
+        acceptScaledBidOffer(
+          seat,
+          bidScaling,
+          want,
+          trySettle,
+          exitAfterBuy = false,
```

Comment:
with two booleans now, please make it an opts object. it's too easy to get the order wrong and nothing would complain

Reply:
done